docs/api.md

```diff
@@ -91,10 +91,10 @@ Type: Array\<[Mixin](#mixins-mixin)\>
 Type: ([props](#app-props)): [props](#mixin-props)
```

Comment:
Should read:

Type: ([props](#app-props), [emit](#emit)): [props](#mixin-props)